draft-sharabayko-mops-srt.md

```diff
@@ -1962,10 +1962,16 @@ have no chance to be delivered in time, and allows the receiver to skip missing
 that have not been delivered in time. The timeout of dropping a packet is based on the 
 TSBPD mechanism ({{tsbpd}}).
 
-In the SRT, when Too-Late Packet Drop is enabled, and a packet timestamp is older than 
-125% of the SRT latency, it is considered too late to be delivered and may be dropped
-by the sender. However, the sender keeps packets for at least 1 second in case the
-SRT latency is not enough for a large RTT (that is, if 125% of the SRT latency is less 
+When TLPKTDROP is enabled, a packet timestamp is older than TLPKTDROP_THRESHOLD,
+the packet is considered "Too-Late" to be delivered. Thus, the sender might need to drop
+the packet.
+
+TLPKTDROP_THRESHOLD: a marginal value that is considered that a packet is delivered in time. 
+: The value is related to SRT Latency({{srt-latency}}). When considering RTT, it recommends
+  1.25 times of SRT Latency.
```

Comment:
Maybe add some note that the value should be that high to allow the receiver to drop the packet first, so that sender drops packets only when serious congestion happens and it does not get a response from the peer.

Reply:
That's a very useful note.